gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -148,6 +149,10 @@ fn v1_routes(gateway: Gateway) -> Router {
         .route("/leave-fed", post(leave_fed))
         .route("/backup", post(backup))
         .route("/restore", post(restore))
+        .route("/connect_to_peer", post(connect_to_peer))
+        .route("/get_funding_address", get(get_funding_address))
```

Comment:
`get_funding_address` should be a `POST` request too since it causes mutation of the service's internal state and should not be cached. One could argue that it always returns *some* address belonging to the node, so is kinda idempotent and can be `GET`, but `POST` feels better.

Reply:
Good point! Done.